ReactiveCocoaTests/InterceptingSpec.swift

```diff
@@ -39,6 +39,26 @@ class InterceptingSpec: QuickSpec {
 				expect(object.counter) == 2
 			}
 
+			it("should send a value when the selector is invoked without implementation") {
+				let selector = #selector(TestProtocol.optionalMethod)
+
+				let signal = object.reactive.trigger(for: selector,
+				                                     from: TestProtocol.self)
+				expect(object.responds(to: selector)) == true
+
+				var counter = 0
+				signal.observeValues { counter += 1 }
+
+				expect(counter) == 0
+
```

Comment:
It does not crash because RAC adds an implementation for it at the time you swizzle it via ` trigger(for:from:)` with the protocol specified.

If you do not swizzle it, it crashes.